brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-638 Fix: Multi Buttons - WPML support not working.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-638

### Screenshots

- https://share.getcloudapp.com/6quz50Nw

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Install WPML > Select Language of translation.
- Create a page with Multi-buttons.
- Click on Translate to hindi ( You can choose your selected language )
- It opens WPML Translation Editor and In which you will see Buttons Text at Left and on right you need to write a text in which you wanted to translate the text I. e Hindi.
- See screencast - https://share.getcloudapp.com/6quz50Nw

### Checklist:
